### ISSUE TITLE
Submissions for certain form not accessible in the CMS

### ISSUE THREAD

**Issue by havardlj** [verbb/formie#584]
I get an error when trying to access submissions from one of the several forms when logged into the CMS. The rest are working just fine. I can also access sub-pages of the submission, just not the overview (with the graph)

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.16
- Craft version: 3.7.13
- Multi-site: No

**Additional context**

Error log:
```
2021-09-21 16:14:17 [-][6][-][error][Error] Error: Object of class DateTime could not be converted to string in /home/forge/mywebsite.no/vendor/craftcms/cms/src/base/Field.php:537
Stack trace:
#0 /home/forge/mywebsite.no/vendor/craftcms/cms/src/base/Element.php(3770): craft\base\Field->getTableAttributeHtml()
#1 /home/forge/mywebsite.no/vendor/verbb/formie/src/elements/Submission.php(932): craft\base\Element->tableAttributeHtml()
#2 /home/forge/mywebsite.no/vendor/craftcms/cms/src/base/Element.php(3646): verbb\formie\elements\Submission->tableAttributeHtml()
#3 /home/forge/mywebsite.no/vendor/twig/twig/src/Extension/CoreExtension.php(1511): craft\base\Element->getTableAttributeHtml()
#4 /home/forge/mywebsite.no/vendor/craftcms/cms/src/helpers/Template.php(106): twig_get_attribute()
#5 /home/forge/mywebsite.no/storage/runtime/compiled_templates/77/77c20b6e1a7cb5ee142a9c6cbb97be88ada8f979c72611d91c2cc0cf80ed2b51.php(167): craft\helpers\Template::attribute()
#6 /home/forge/mywebsite.no/vendor/twig/twig/src/Template.php(405): __TwigTemplate_bb2ff97a42b5deacf154d756a45517f17c6b3fcb56e7a53486ca4e67301f4224->doDisplay()
#7 /home/forge/mywebsite.no/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#8 /home/forge/mywebsite.no/storage/runtime/compiled_templates/65/65084a6af385eca5dcc37a5039b91c01503d2bdc32ad2475cd8a9e48bd0e5042.php(102): Twig\Template->display()
#9 /home/forge/mywebsite.no/vendor/twig/twig/src/Template.php(405): __TwigTemplate_c3522692dd5474c9950d78ed55dde7b46e6c52e23080a827ecceef8d47f51e14->doDisplay()
#10 /home/forge/mywebsite.no/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#11 /home/forge/mywebsite.no/vendor/twig/twig/src/Template.php(390): Twig\Template->display()
#12 /home/forge/mywebsite.no/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render()
#13 /home/forge/mywebsite.no/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render()
#14 /home/forge/mywebsite.no/vendor/craftcms/cms/src/web/View.php(408): Twig\Environment->render()
#15 /home/forge/mywebsite.no/vendor/craftcms/cms/src/base/Element.php(793): craft\web\View->renderTemplate()
#16 /home/forge/mywebsite.no/vendor/craftcms/cms/src/controllers/ElementIndexesController.php(534): craft\base\Element::indexHtml()
#17 /home/forge/mywebsite.no/vendor/craftcms/cms/src/controllers/ElementIndexesController.php(135): craft\controllers\ElementIndexesController->elementResponseData()
#18 [internal function]: craft\controllers\ElementIndexesController->actionGetElements()
#19 /home/forge/mywebsite.no/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#20 /home/forge/mywebsite.no/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams()
#21 /home/forge/mywebsite.no/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction()
#22 /home/forge/mywebsite.no/vendor/yiisoft/yii2/base/Module.php(534): craft\web\Controller->runAction()
#23 /home/forge/mywebsite.no/vendor/craftcms/cms/src/web/Application.php(277): yii\base\Module->runAction()
#24 /home/forge/mywebsite.no/vendor/craftcms/cms/src/web/Application.php(586): craft\web\Application->runAction()
#25 /home/forge/mywebsite.no/vendor/craftcms/cms/src/web/Application.php(256): craft\web\Application->_processActionRequest()
#26 /home/forge/mywebsite.no/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest()
#27 /home/forge/mywebsite.no/web/index.php(21): yii\base\Application->run()
#28 {main}
```

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.18](https://github.com/verbb/formie/releases/tag/1.4.18)
